### DISCUSSION TITLE
Maya Arnold Renderer Variant Support

### DISCUSSION THREAD

**Discussion by brycegbrazen** [Autodesk/arnold-usd#890] [Q&A] [answered]
I see that variants are changeable in Maya as of Maya 2022 and I can see the variant changes in Viewport 2.0, but it appears that the variant changes do not apply to the render output. Does Arnold support variants in the render output somehow?

Update: I see that this is an issue with mayaUSD v0.12.0 which uses USD 21.05. The issue seems to stem from the fact that arnold-usd and hydra do not yet support USD 21.05. However, the default mayaUSD installed with Maya 2022 (v0.10.0) seems to work fine with variants. Seems like we just need a new arnold-usd release.

**Comment by sebastienblor** [comment, marked as answer]
Hi @brycegbrazen , sorry for the lack of reply, we had missed this section in github.
Glad to hear that it's solved
